cartoframes/data/observatory/repository/repo_client.py

```diff
@@ -2,15 +2,14 @@
 
 from cartoframes.data.clients import SQLClient
 from cartoframes.auth import Credentials
-from .constants import DO_METADATA_KEY
 
 
 class RepoClient(object):
 
     __instance = None
 
     def __init__(self):
-        self.client = SQLClient(Credentials('do-metadata', DO_METADATA_KEY))
+        self.client = SQLClient(Credentials('do-metadata', 'default_public'))
 
     def get_countries(self, field=None, value=None):
         query = 'select distinct country_iso_code3 from datasets_public'
```

Comment:
Can we add here `country_iso_code3 as id` to standardize all the fields are ids?

Reply:
The Country class has that field exposed as "id", so the user will see it like that. I let the raw data untouched to preserve the original name so we keep consistency with the repo at the lowest layer.

Reply:
Nah, I just change id since it's just a field in the dataset table, not a table on its own :)